docs/data/migration/migration-data-grid-v5/migration-data-grid-v5.md

```diff
@@ -33,9 +31,9 @@ You can either run it on a specific file, folder, or your entire codebase when c
 
 ```bash
 // Data Grid specific
-npx @mui/x-codemod v6.0.0/data-grid/preset-safe <path>
+npx @mui/x-codemod@latest v6.0.0/data-grid/preset-safe <path>
```

Comment:
See https://stackoverflow.com/questions/73184553/npx-does-not-look-for-latest-version-of-package for why. We faced this with Toolpad. Also why Next.js has `@latest` in https://nextjs.org/